integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeImplementationIT.java

```diff
@@ -1,147 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package org.opensearch.sql.ppl;
-
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_DATE;
-import static org.opensearch.sql.util.MatcherUtils.rows;
-import static org.opensearch.sql.util.MatcherUtils.schema;
-import static org.opensearch.sql.util.MatcherUtils.verifySchema;
-import static org.opensearch.sql.util.MatcherUtils.verifySome;
-
-import java.io.IOException;
-import org.json.JSONObject;
-import org.junit.Test;
-
-public class DateTimeImplementationIT extends PPLIntegTestCase {
```

Comment:
Revert this too

Reply:
reverted in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0